wallet/walletrpc.c

```diff
@@ -146,6 +146,32 @@ static struct command_result *broadcast_and_wait(struct command *cmd,
 	return command_still_pending(cmd);
 }
 
+/* Now we only support using `txprepare` for "withdrawal",
+ * "funding" and "unknown". */
+static struct command_result *param_txtype(struct command *cmd,
+					   const char *name,
```

Comment:
In fact, the code here is aligned. Maybe it's the display error?

Reply:
It has to do with tabs.  Tabs are aligned to 8 positions.  Try typing a line with tab first then nonwhitespace character, then on next line type a nonwhitespace character, then tab, then nonwhitespace character.

You can ignore this for the most part.

Reply:
Other than that... I am tempted to just make `txtype` an arbitrary string, just change the db schema instead.  Though I am still not quite up-to-date to the latest db changes.

Reply:
Like the `label` for `invoice`?

Reply:
not quite, `label` for `invoice` has to be unique per invoice.

Reply:
Yeah, it need to replace `enum wallet_tx_type` with a string field.
Should this change deserve another PR?

Reply:
Maybe same PR? Depends on what others think though.

Reply:
This change seems be related with `listtransactions`(#2696 ). I will continue to study it.